Fix broken link on Installing Addons page page (#37749)
kubernetes/website: content/en/docs/concepts/cluster-administration/addons.md

## Visualization &amp; Control

* [Dashboard](https://github.com/kubernetes/dashboard#kubernetes-dashboard) is a dashboard web interface for Kubernetes.

> * [Weave Scope](https://www.weave.works/docs/scope/latest/) is a tool for visualizing your containers, Pods, Services and more.

Reviewer: As this article intend to install plugins i think this would be a better reference to it https://www.weave.works/docs/scope/latest/installing/

Author: Hello @reylejano @divya-mohan0209 I believe the [link](https://www.weave.works/docs/scope/latest/installing/) section is mentioned on this [page](https://www.weave.works/docs/scope/latest/), Please suggest what changes need to be made.

Reviewer 2: Hello @kadtendulkar .
The existing link (`https://www.weave.works/documentation/scope-latest-installing/#k8s`) redirects to `https://www.weave.works/docs/scope/latest/installing/#k8s`, similar to the link suggested by @Kartik494 .

Do you want to use this link (`https://www.weave.works/docs/scope/latest/installing/#k8s`)?

> * [Weave Scope](https://www.weave.works/documentation/scope-latest-installing/#k8s) is a tool for graphically visualizing your containers, pods, services etc. Use it in conjunction with a [Weave Cloud account](https://www.weave.works/blog/weave-cloud-end-of-service) or host the UI yourself.

Reviewer 3: ```suggestion
* [Weave Scope](https://www.weave.works/docs/scope/latest/) is a tool for visualizing your containers, pods, services and more.
```

Reviewer 4: +1 to sftim's suggestion
